src/custom/pages/Swap/index.tsx

```diff
@@ -47,34 +47,52 @@ const SwapModWrapper = styled(SwapMod)`
       grid-row-gap: 3px;
     }
 
+    .expertMode ${AutoColumn} {
+      grid-row-gap: 12px;
+    }
+
+    ${ClickableText} {
+      color: ${({ theme }) => theme.text1};
+    }
+
     ${Card} > ${AutoColumn} {
       margin: 6px auto 0;
     }
 
     ${ArrowWrapperUni} {
       position: absolute;
       z-index: 2;
-      background: ${({ theme }) => theme.white};
-      border-radius: 9px;
-      width: 28px;
-      height: 28px;
+      background: ${({ theme }) => theme.swap?.arrowDown?.background};
```

Comment:
why are all these conditional?

Reply:
Made them conditional as it would leave out this property, if the `swap` and `arrowDown` are not provided in the theme. Which in case it falls back to the original (mod) properties (no override in that case was my thinking).

Reply:
you're sure there's a fallback? it also seems a bit unnecessary to make such a specific nested colour. does `arrowDown` have many properties that are unique?

Reply:
Yes there's a fallback. Just updated now with removing the conditionals and these values are now mandatory (moved them to the baseTheme).

The reason for defining these values as theme properties, is that I wanted to make use of the `darkMode` value, without having to bring that over to the individual component. But instead make sure of the `darkMode` value in the theme value. Let me know your thoughts.